client/my-sites/marketplace/components/review-item/index.tsx

```diff
@@ -89,6 +92,23 @@ export const MarketplaceReviewItem = ( props: MarketplaceReviewItemProps ) => {
 		clearEditing();
 	};
 
+	const maybeRenderContent = () => {
+		if ( isEmptyContent ) {
+			return null;
+		}
+
+		return (
+			<div
+				// sanitized with sanitizeSectionContent
+				// eslint-disable-next-line react/no-danger
+				dangerouslySetInnerHTML={ {
+					__html: sanitizeSectionContent( review.content.rendered ),
+				} }
+				className="marketplace-review-item__content"
+			></div>
+		);
+	};
```

Comment:
Non-blocking: As this is rendering something (or null) we may should create a component in this same file with this logic.

Just because in my mind is easier to read `<RenderedReview review={review} />` than `maybeRenderContent()`

Reply:
> That is just a matter of style

Indeed, use what you prefer.